iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergTestUtils.java

```diff
@@ -299,4 +311,68 @@ public static void validateDataWithSQL(TestHiveShell shell, String tableName, Li
       }
     }
   }
+
+  /**
+   * @param table The table to create the delete file for
+   * @param deleteFilePath The path where the delete file should be created, relative to the table location root
+   * @param equalityFields List of field names that should play a role in the equality check
+   * @param fileFormat The file format that should be used for writing out the delete file
+   * @param rowsToDelete The rows that should be deleted. It's enough to fill out the fields that are relevant for the
+   *                     equality check, as listed in equalityFields, the rest of the fields are ignored
+   * @return The DeleteFile created
+   * @throws IOException If there is an error during DeleteFile write
+   */
+  public static DeleteFile createEqualityDeleteFile(Table table, String deleteFilePath, List<String> equalityFields,
```

Comment:
The delete id = 3 could be written as either of the following equality delete files:

equality_ids=[1]

 1: id | -
-------|-
 3     | -        

equality_ids=[1]

Or:

 1: id | 2: category | 3: name
-------|-------------|---------
 3     | NULL        | Grizzly

We might want to test both

Reply:
I think I'm covering that by leaving the last_name field as null, passing this in:
`add(1L, "Bob", null)`
i.e. not filling out the irrelevant fields.
Or is there something different you're looking for?

Reply:
Added a new test which omits a column that is not part of the equality check